### PR TITLE
XS✔ ◾ Made "include URL" rule generic... instead of just for "Dones"

### PR DESCRIPTION
**Pull request by tiagov8**
cc @adamcogan @BenDunkerleySSW 

> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

**Re: Formatting changes to SugarLearning item** 

> 2. What was changed?

Made "include URL" rule generic... instead of just for "Dones"

> 3. Did you do pair or mob programming (list names)?

no

### CODE REVIEW

File: rules/dones-do-your-dones-include-a-url/rule.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 type: rule
-title: Dones - Do your 'Dones' include a URL?
+title: Do you include URLs to tasks and "Done" emails?
```
Comment (jaydenalchin) [inline rules/dones-do-your-dones-include-a-url/rule.md:3]:
```suggestion
title: Do you include URLs in tasks and "Done" emails?
```

File: rules/dones-do-your-dones-include-a-url/rule.md [hunk 2]
```diff
@@ -9,7 +9,9 @@ authors:
     url: https://ssw.com.au/people/cameron-shaw
   - title: Ulysses Maclaren
     url: https://ssw.com.au/people/ulysses-maclaren
-related: []
+related:
+  - add-context-reasoning-to-emails
+  - dones-do-you-include-useful-details-in-your-done-email
```
Comment (jaydenalchin) [inline rules/dones-do-your-dones-include-a-url/rule.md:14]: Is this rule actually a duplicate of dones-do-you-include-useful-details-in-your-done-email ?

**Review by jaydenalchin — CHANGES_REQUESTED** [review]

File: rules/dones-do-your-dones-include-a-url/rule.md (hunk 2 above)
Comment (tiagov8) [reply rules/dones-do-your-dones-include-a-url/rule.md:14]: Hi @jaydenalchin 

I see as:

- https://ssw.com.au/rules/add-context-reasoning-to-emails/
  Add context when you send an email

- https://ssw.com.au/rules/dones-do-you-include-useful-details-in-your-done-email/
  Add any useful info, such as context, screenshots, code changes etc... to "Done" emails 

- https://ssw.com.au/rules/dones-do-your-dones-include-a-url/
  Add URLs (links) to any email
  
  I think the 3 rules can live together... they are out there for a while... but if you disagree... I would recommend getting this PR through and opening a new Issue to look into merging them

**Comment by tiagov8** [comment]
Hi @jaydenalchin 

FYI - I updated the URI for this rule after you previous review/comments

**Review by jaydenalchin — APPROVED** [review]

**Review by jaydenalchin — APPROVED** [review]

**Pull request merged**